docker-compose.yml

```diff
@@ -62,7 +62,7 @@ services:
 
   # OpenMRS
   openmrs:
-    image: mekomsolutions/openmrs:latest
+    image: mekomsolutions/openmrs:69f1dce
```

Comment:
Taking this one as an example, is there no way to not use hashes but rather a user-friendly version (like it's done for [`openmrs/openmrs-reference-application-3-frontend:3.0.0-beta.2`](https://github.com/ozone-his/ozone-docker/pull/28/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R189))?